oozie/oozie.sh

```diff
@@ -203,6 +205,39 @@ function configure_ssl() {
   set_oozie_property 'oozie.https.truststore.file' "${truststore_file}"
 }
 
+function get_file() {
+  url=$1
+  file=$2
+  dst=$3
+  if [ -n "${OOZIE_REPO_URL}" ]; then
+    gsutil cp ${OOZIE_REPO_URL}/$file "$dst"
+  else
+    wget -nv --timeout=30 --tries=5 --retry-connrefused \
+      $url/$file -P "$dst"
+  fi
+}
+
+function dnf_install() {
+  pkg=$1
+  if [ -n "${OOZIE_REPO_URL}" ]; then
+    gsutil cp "${OOZIE_REPO_URL}/$pkg*" "/tmp/fname.rpm"
```

Comment:
Please convert `fname.rpm` to a random file name (add timestamp, uuid, etc) to avoid potential collusions.

Reply:
Solved.